src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -1577,6 +1662,40 @@ class OverlapsClauseSegment(BaseSegment):
     match_grammar = Nothing()
 
 
+@tsql_dialect.segment(replace=True)
+class DescribeStatementSegment(BaseSegment):
+    """A `Describe` statement.
+
+    Not present in T-SQL.
+    """
+
+    type = "describe_statement"
+    match_grammar = Nothing()
+
+
+@tsql_dialect.segment(replace=True)
+class CommentClauseSegment(BaseSegment):
+    """A comment clause.
+
+    Not present in T-SQL.
+    """
+
+    type = "comment_clause"
+    match_grammar = Nothing()
+
+
+@tsql_dialect.segment(replace=True)
+class MLTableExpressionSegment(BaseSegment):
+    """An ML table expression.
+
+    Not present in T-SQL.
+    TO DO: Consider whether this segment can be used to represent a PREDICT statement.
+    """
+
+    type = "ml_table_expression"
+    match_grammar = Nothing()
```

Comment:
This ones only referenced in `FromExpressionSegment` in ANSI:
![image](https://user-images.githubusercontent.com/80432516/147369001-1467e6fe-086a-42ad-b61a-ca33ae0f92ab.png)

It's fairly simple so I guess maybe we could just copy this to TSQL but without MLTableExpressionSegment?

Happy to leave as a stub for now whilst you investigate the PREDICT statement implementation 👍

Reply:
@jpy-git I've added issue #2199 to cover the PREDICT implementation.

Reply:
thanks! 👍